known_friends.md

```diff
@@ -0,0 +1,8 @@
+# Known Friends of EDC
+
+To get to know how we define "adoptions" and see how to submit a feature, please take a look at our
+[guidelines for submitting features](../submodule/Connector/contribution_categories.md).
+
+| Title | Description | Contact | Links
+| :--- | :--- | :--- | :--- 
+| _My awesome project_ | _This is an example._ | _e-mail-address_ | _link to repository, homepage, discussion, etc._
```

Comment:
It's a little sad to have a "known friends" page actually empty :laughing: 
Could we add at least Catena-X, Eona-X, ...?

Reply:
Maybe @mspiekermann can fill this list with the right links (repos, projects) and contact persons. Just writing "C-X" and "Eona-X" will not be that informative. I will create an issue for this. As we don't have this list right now, the content is not needed for this PR. Most important is, that this file exists and can be linked